web/Content/img/logo-header.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="174" height="38" viewBox="0 0 174 38" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All added files must be specified in the `ASC.Api.Web.Help.csproj`, otherwise they will be lost when publishing.